vagrant-spk

```diff
@@ -291,7 +291,13 @@ def call_vagrant_command(sandstorm_dir, *command_args):
     command = ["vagrant"]
     command.extend(command_args)
     sys.stderr.write("Calling {} in {}\n".format(" ".join(["'{}'".format(arg) for arg in command]), sandstorm_dir))
-    return subprocess.check_call(command, cwd=sandstorm_dir)
+    try:
+        return subprocess.check_call(command, cwd=sandstorm_dir)
+    except subprocess.CalledProcessError as e:
+        ANSI_RED = "\x1b[31m"
+        ANSI_RESET = "\x1b[0m"
+        msg = "Command failed with a non-zero exit status (%d)." % e.returncode
```

Comment:
nit: generally I prefer the [new-style format strings](https://docs.python.org/2/library/string.html#format-string-syntax) over printf-style `%` formatting